uint32 mode
string node
---
string result
